src/oidcop/oidc/token.py

```diff
@@ -81,12 +81,9 @@ def process_request(self, req: Union[Message, dict], **kwargs):
 
         logger.debug("All checks OK")
 
-        issue_refresh = False
-        if "issue_refresh" in kwargs:
-            issue_refresh = kwargs["issue_refresh"]
-        else:
-            if "offline_access" in grant.scope:
-                issue_refresh = True
+        issue_refresh = kwargs.get("issue_refresh", False)
+        if "offline_access" in grant.scope:
+            issue_refresh = True
```

Comment:
The logic is reversed here as well. `issue_refresh` used to override `offline_access`, which IMHO is the correct behavior. Maybe this is better:
```python
        issue_refresh = kwargs.get("refresh_token", None)
        # The existence of offline_access scope overwrites issue_refresh
        if issue_refresh is None and "offline_access" in scope:
            issue_refresh = True
```

Reply:
ok @nsklikas I wait for your revision before merge